### ISSUE TITLE
Add a focus ring arround the linkarea

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#7000]
When the `post-linkarea` has a `:focus-within` it should have a visible focus ring. 
The link itself should not have a focus ring.
